### PR TITLE
Changes kick, walk, strat, planning

### PR DESCRIPTION
**Pull request by ysims**
Updates teen sized scripts/config for LeftKick, RightKick, Stand, OldWalkEngine. Updates SoccerStrategy,  SimpleWalkPathPlanner to reflect the limitations of the walk engine.

### CODE REVIEW

File: module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ start_position_offensive: [-0.2, 0]
 start_position_defensive: [1.5, 0]
 goalie: false
 
-ball_search_walk_start_speed: 0.5
-ball_search_walk_stop_speed: 0.1
+ball_search_walk_start_speed: 0.0
+ball_search_walk_stop_speed: 0.0
```
Comment (MattAmos) [inline module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml]: Not sure if we should do this, should test to see how a 0 velocity impacts the system

Comment (MattAmos) [inline module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml]: Not sure if we should do this, I'm not sure how the system would handle a 0 velocity

File: module/motion/OldWalkEngine/data/config/nugus2/OldWalkEngine.yaml [hunk 2]
```diff
@@ -63,22 +63,22 @@ walk_cycle:
   velocity:
     # Velocity limits for the walk
     limits:
-      - [-0.0027, 0.3] # [min_x, max_x]
+      - [-0.2, 0.2] # [min_x, max_x] #-0.0027
```
Comment (MattAmos) [inline module/motion/OldWalkEngine/data/config/nugus2/OldWalkEngine.yaml]: Remove this comment

File: module/motion/OldWalkEngine/data/config/nugus2/OldWalkEngine.yaml [hunk 3]
```diff
@@ -27,31 +27,31 @@ stance:
 
   foot_offset:
     - -0.05 # The distance the ankles are offset from the torso, in meters.
-    - -0.07 # How far from the center of the foot the center of mass is placed during each step.
+    - -0.1 # How far from the center of the foot the center of mass is placed during each step.
   # The gains to use for the servos during the walk
   gains:
-    arms: 15
+    arms: 0 # 15
```
Comment (MattAmos) [inline module/motion/OldWalkEngine/data/config/nugus2/OldWalkEngine.yaml]: Remove this comment

File: module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml (hunk 1 above)
Comment (MattAmos) [inline module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml]: Not sure if we should do this, I'm not sure how the system would handle a 0 velocity

Comment (ysims) [reply module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml]: can confirm it worked with 0 :P

**Comment by ysims** [comment]
Is buildkite broken?

**Comment by ysims** [comment]
Old, out of date. QuinticWalk will need new configs.

**Pull request closed without merging**